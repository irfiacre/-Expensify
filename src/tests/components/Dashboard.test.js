import React from 'react';
import { shallow } from 'enzyme';
import DashBoard from '../../components/dashboard';

describe('*************** Testing the DashBoard  component ***************',()=>{
    test('Should render the DashBoard  page correctly', ()=>{
        const wrapper = shallow( <DashBoard  /> );
        expect(wrapper).toMatchSnapshot();
    });

});