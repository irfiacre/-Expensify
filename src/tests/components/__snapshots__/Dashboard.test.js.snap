// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`*************** Testing the DashBoard  component *************** Should render the DashBoard  page correctly 1`] = `
<div>
  <div
    className="contentContainer"
  >
    <h2
      className="dashboardTitle"
    >
      Sort By
    </h2>
  </div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpensesList) />
</div>
`;
